.github/workflows/deploy_backend_prod.yml

```diff
@@ -0,0 +1,95 @@
+name: Deploy BACKEND to VPS (prod)
+
+on: 
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+env:
+  BUILD_PATH: ${{ secrets.PROD_PATH_TO_DEPLOY }}/backend
+  BRANCH: main
+
+jobs:
+  Update-server:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull changes in dev server
+        uses: appleboy/ssh-action@v0.1.10
+        with:
+          host: ${{ secrets.PROD_SSH_HOST }}
+          username: ${{ secrets.PROD_SSH_USERNAME }}
+          password: ${{ secrets.PROD_SSH_PASSWORD }}
+          port: ${{ secrets.PROD_SSH_PORT }}
+          script: |
+            cd ${{ secrets.PROD_PATH_TO_DEPLOY }}
+            branch=$(git rev-parse --abbrev-ref HEAD)   # chech current branch
+            echo "Current branch is $branch"
+            if [[ "$branch" != "${{ env.BRANCH }}" ]]; then           
+              git checkout ${{ env.BRANCH }}
+            fi
+            git fetch --all
+            git reset --hard origin/${{ env.BRANCH }}
+            git pull
+
+      - name: Install deps in dev server
+        uses: appleboy/ssh-action@v0.1.10
+        with:
+          host: ${{ secrets.PROD_SSH_HOST }}
+          username: ${{ secrets.PROD_SSH_USERNAME }}
+          password: ${{ secrets.PROD_SSH_PASSWORD }}
+          port: ${{ secrets.PROD_SSH_PORT }}
+          script: |
+            cd ${{ secrets.PROD_PATH_TO_DEPLOY }}/backend
+            export NVM_DIR=~/.nvm
+            source ~/.nvm/nvm.sh
+            npm install
+
+  build-backend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
```

Comment:
The latest version of strapi requires nodejs 18
I would advise to upgrade to 18